### ISSUE TITLE
Training for Font Restyle 

### ISSUE THREAD

**Issue by anas-899** [yuval-alaluf/hyperstyle#8]
Thanks for sharing another amazing work of yours

I am wondering if i can train it for another domain such as font style

I have followed some of the steps of facebook paper:
TextStyleBrush: Transfer of Text Aesthetics from a Single Example
https://arxiv.org/pdf/2106.08385
By training stylegan2 with some additions for non-square images and loss function.
I got good results but of course far from what facebook are getting in their paper.
However the biggest problem was the inversion of the image by projection to get the latent code. It was very slow.

Do you think this or psp can work for another domain such as font restyle?

**Comment by yuval-alaluf** [comment]
If you have the pretrained SG2 trained on your domain, I believe that this can help, especially with respect to the inference time. 
I would start off by trying to train either HyperStyle or ReStyle (probably applied over e4e), which should be stronger than trying working only with the standard pSp encoder. 
As a sanity check, I would try overfitting on a small number of samples to see if the model has the capacity to learn the details you're looking for.
